### DISCUSSION TITLE
Are (or were) Dask docker containers affected by log4j?

### DISCUSSION THREAD

**Discussion by vecorro** [dask/dask#8701] [Q&A]
I got this question from my infosec team and I wonder if any verifications have been made on the Docker containers that get deployed when the Dask helm char is used to deploy new clusters. Thanks!

**Comment by jsignell** [comment]
Ping @jacobtomlinson who is the resident docker expert :)

**Comment by jacobtomlinson** [comment]
**TL;DR** I don't know.

The only component of Dask that uses Java is `dask-sql` and we do not install that in the container by default. 

We don't do any intentional security scanning on our Docker containers. We do build them regularly as Dask releases every couple of weeks, so up to date containers should have up to date patches. But we do not verify this.
